blast/formatter/server/internal/server/run-report-ep.go

```diff
@@ -100,11 +112,21 @@ func zipDir(path string) (err error) {
 
 	var tf io.Writer
 	for i := range matches {
-		tf, err = zp.Create(filepath.Base(matches[i]))
+
+		baseFileName := filepath.Base(matches[i])
+
+		// This should never happen given the check above, but just to be safe!
+		if baseFileName == reportExportName {
+			return errors.New(baseFileName + " already exists!")
```

Comment:
Where does this error get propagated to? Is there any value in distinguishing the error message in this case as opposed to above case to help debug the race condition?

Reply:
The error is passed up to the caller that throws a 500 error if `err != nil`